### ISSUE TITLE
Remove/Delete menu entry for the track in playlist

### ISSUE THREAD

**Issue by slrslr** [audacious-media-player/audacious#1516]
**REMOVE:**

Please add a "Remove" right-mouse-click menu entry to remove a track from the playlist.
Other similar track manipulation menu items (like copy/cut) exist already. Remove item is important one that is used by me more often than any other when processing the playlist with a new songs.

Currently I have to use Delete key on the keyboard, which is not always handy comparing to mouse, which i already have pointed on the track.

**DELETE:**

It may be also handy to have "Delete" menu entry (near the "Remove" one) which would ask for a confirmation to delete actual audio file from a drive (possibly use some WARNING!!! style not to confuse a file deletion with a playlist removal). Deletion is claimed to already exist, but through the plugin https://github.com/audacious-media-player/audacious/issues/1266 which I assume is not enabled by default.

* Do other audio players offer this feature?

Yes, All good players that I have used (like MPC-HC, Clementine, WinAmp) allows removal when one clicks an item.

**Comment by mschwendt** [comment]
Why not simply use "Cut" which is readily available? It's also highly beneficial to prefer using "Cut", since it serves as level one "Undo" in case you made a mistake and wanted to paste the removed entry into the playlist again or paste its filename somewhere else.

"Remove" is as ambiguous as "Delete" in my opinion.

**Comment by slrslr** [comment]
No, to me Cutting does not suggests removal (even it works like that), but from a operating system principles it instead suggests a two-step action which is not completed if second step is not done, and when wanting to Remove, i do not want to do that second step (pasting to some other playlist). Though I can see that Cutting indeed works as a removal (second step is not needed).

> ambiguous

that is why the confirmation prompt in case of a delete, for those that does not consider delete as permanent and remove as a "soft" action.

**Comment by slrslr** [comment]
Not in my experience. In Debian KDE main file manager Dolphin and on Windows explorer is required 2-steps AFAIK.

**Comment by mschwendt** [comment]
That's a safety measure to avoid deleting files accidentally, if cutting files without pasting them anywhere. Moving them to a wastebasket folder temporarily would not be as safe.

I'd rather compare editing a playlist to editing a text file. Cutting some text with a text editor deletes the text immediately. Most text editors can undo/redo a few levels of general editing, though.
